client/src/FilterControl.jsx

```diff
@@ -10,7 +10,7 @@ const FilterControl = props => {
     filtering = Map(), toggleMainCharFilter, toggleVsCharFilter, toggleStageFilter, toggleStandaloneTagFilter,
   } = props;
   return (
-    <Panel bsStyle="primary" header={<h2> Filtering Options </h2>}>
+    <Panel variant="primary" header={<h2> Filtering Options </h2>}>
```

Comment:
Wait so do we need this and the TS version?

Reply:
Removed.